labshare-client/src/views/Index.vue

```diff
@@ -1,9 +1,9 @@
 <i18n>
     {
     "en":{
-    "mainTitle": "We combine forces for more SARS-CoV-2 tests",
-    "subTitle": "Together we unleash the Power of Science!",
-    "Joinbutton": "Join now!",
+    "0title": "We combine forces for more SARS-CoV-2 tests",
+    "1title": "Together we unleash the cumulative Power of Science!",
```

Comment:
"power" should be lowercase, but otherwise nothing stands out.

Reply:
I think it's more meant as a proper noun.